Soccar [Library]/src/nl/soccar/library/enumeration/SteerAction.java

```diff
@@ -7,6 +7,31 @@
  */
 public enum SteerAction {
     
-    NONE, STEER_LEFT, STEER_RIGHT
+    NONE(0),
+    STEER_LEFT(1),
+    STEER_RIGHT(2);
+    
+    private final int id;
+    
+    private SteerAction(int id) {
+        this.id = id;
+    }
+    
+    public int getId() {
+        return id;
+    }
+    
+    public static SteerAction parse(int id) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)